build.xml

```diff
@@ -0,0 +1,1075 @@
+<!--

```

Comment:
Why is there a build.xml file here?  I assume this was a mistake to checkin.

Reply:
yup ... removed